test/test_icalevents.py

```diff
@@ -107,6 +107,32 @@ def test_events_all_day_recurring(self):
         self.assertEqual(ev_2.description, "All-day event recurring on tuesday each week")
         self.assertTrue(ev_2.all_day, "Recurring All-day Event's second instance is an all-day event")
 
+    def test_events_rrule_until_all_day_ms(self):
+        ical = "test/test_data/rrule_until_all_day_ms.ics"
+        start = date(2021, 1, 1)
+        end = date(2022, 1, 1)
+
+        evs = icalevents.events(file=ical, start=start, end=end)
+        ev_0 = evs[0]
+
+        self.assertEqual(len(evs), 6, "Seven events and one is excluded")
```

Comment:
Why is it excluded? Maybe add a comment explaining this. 
If it Is transparent opaque add that here so it reads something like
"Seven events and one transparent opaque excluded"

Reply:
it is an EXDATE - exclusion date :smile: